packages/styles/scss/fonts/__tests__/fonts-test.js

```diff
@@ -91,6 +96,56 @@ describe('@carbon/styles/scss/fonts', () => {
     expect(emitted.get('IBM Plex Serif').styles).toEqual(
       new Set(['normal', 'italic'])
     );
+
+    expect(emitted).toMatchSnapshot();
+  });
+
+  it('should emit src from akamai cdn if $use-akamai-cdn is true', async () => {
+    const { result } = await render(`
+      @use '../../config' with (
+        $use-akamai-cdn: true,
+      );
+      @use '../' as fonts;
+    `);
```

Comment:
@aledavila the tests actually already configure the flag on - the output is in the `.snap` files.

So from a testing perspective I think it's just making sure that the snapshots make sense and the links resolve

Reply:
You totally could turn the flag on globally though in your local environment and load up storybook as a smoke test to see if everything looks right and the akamai assets are being pulled in.

Reply:
Thanks Taylor. That’s easier let me check that